math/vecops/src/RStrongBool.cxx

```diff
@@ -0,0 +1,34 @@
+// Author: Danilo Piparo CERN  06/2018
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "ROOT/RStrongBool.hxx"
+
+#include <string>
+
+namespace ROOT {
+namespace VecOps {
+
+RStrongBool::RStrongBool() : fVal(false){};
```

Comment:
why having a default value at all? would the removal of a default ctor make it awkward to use?